src/tooling/simulating/qemu.md

```diff
@@ -23,13 +23,13 @@ If you prefer to use [`espflash`][espflash], you can achieve the same result by
 
 ```shell
 cargo build --release
-espflash save-image --chip ESP32 --merge target/xtensa-esp32-espidf/release/<NAME> <OUTFILE>
+espflash save-image --chip esp32 --merge target/xtensa-esp32-espidf/release/<NAME> <OUTFILE>
 ```
 
 Now, run the image in QEMU:
 
 ```shell
-/path/to/qemu-system-xtensa -nographic -machine esp32 -drive file=<OUTFILE>,if=mtd,format=raw
+/path/to/qemu-system-xtensa -nographic -machine esp32 -drive file=<OUTFILE>,if=mtd,format=raw -m 4M
```

Comment:
What does the `-m 4M` argument does? Flash size?

Reply:
It allows using external PSRAM in the firmware. See https://github.com/espressif/esp-toolchain-docs/blob/main/qemu/esp32/README.md#adding-psram

Reply:
Is that something that we would like to have included in the command that we showcase? We already have a link to the `esp-toolchain-docs` also, so user can check it and see the available args.

Reply:
I thought think it's very common to use external RAM, this makes it easier to get started.